iotdb-core/metrics/interface/src/main/java/org/apache/iotdb/metrics/metricsets/jvm/JvmGcMetrics.java

```diff
@@ -116,17 +163,67 @@ public void bindTo(AbstractMetricService metricService) {
             String gcAction = notificationInfo.getGcAction();
             GcInfo gcInfo = notificationInfo.getGcInfo();
             long duration = gcInfo.getDuration();
+
+            // The duration supplied in the notification info includes more than just
+            // application stopped time for concurrent GCs (since the concurrent phase is not
+            // stop-the-world).
+            // Try and do a better job coming up with a good stopped time
+            // value by asking for and tracking cumulative time spent blocked in GC.
+            if (isPartiallyConcurrentGC(mbean)) {
+              long previousTotal = lastGcTotalDuration.get();
+              long total = mbean.getCollectionTime();
```

Comment:
If there are 2 GCS, and only 1 meets the requirement of 'isPartiallyConcurrentGC', will the result be twice as large?